src/StringUtils.cpp

```diff
@@ -236,7 +236,7 @@ AString TrimString(const AString & str)
 	size_t end = len;
```

Comment:
Could just as well be `size_t end = len - 1;` to avoid unnecessarily comparing the null character at the end of the string. 
http://www.cplusplus.com/reference/string/string/operator%5B%5D/